src/Nest/Ingest/Processors/DissectProcessor.cs

```diff
@@ -73,6 +73,7 @@ public class DissectProcessorDescriptor<T>
 		public DissectProcessorDescriptor<T> Pattern(string pattern) =>
 			Assign(pattern, (a, v) => a.Pattern = v);
 
+		// TODO! rename the parameter in 8.0
```

Comment:
I think we can rename this here. I understand that it is technically a binary breaking change

Reply:
Let's rename in 8.0 instead, eventhough impact is small.

Reply:
ok